tests/functions/module_lookup.test.ts

```diff
@@ -954,3 +954,51 @@ describe("XLOOKUP formula", () => {
     });
   });
 });
+
+describe("INDEX formula", () => {
+  test("Check argument validity", () => {
+    expect(evaluateCell("A1", { A1: "=INDEX()" })).toBe("#BAD_EXPR");
+    expect(evaluateCell("A1", { A1: "=INDEX(B1:C5, 'string')" })).toBe("#BAD_EXPR");
+    expect(evaluateCell("A1", { A1: "=INDEX(B1:C5, -1)" })).toBe("#BAD_EXPR");
+    expect(evaluateCell("A1", { A1: "=INDEX(B1:C5, , -1)" })).toBe("#ERROR");
+  });
+
+  test("Check row/col index in given range", () => {
+    expect(evaluateCell("A1", { A1: "=INDEX(B1:C5, 6)" })).toBe("#BAD_EXPR");
+    expect(evaluateCell("A1", { A1: "=INDEX(B1:C5, , 4)" })).toBe("#ERROR");
+  });
+
+  test("Select single cell", () => {
+    //prettier-ignore
+    const grid = evaluateGrid({
+      A1: "3",    B1: "Hello",  C1: "=SUM(A1:A3)",
+      A2: "333",  B2: "World",  C2: "=GT(A3,A1)",
+      A3: "33",   B3: "!",      C3: "=CONCAT(B3,B2)",
+      A5: "=INDEX(A1:A3, 1, 1)",
+      B5: "=INDEX(B1:B3, 3, 1)",
+      C5: "=INDEX(C1:C3, 2, 1)",
+      D5: "=INDEX(A1:A3, 3, 1)",
+      E5: "=INDEX(A1:A3, 0, 0)",
+      F5: "=INDEX(A1:A3, 4, 1)",
+      G5: "=INDEX(A1:A3, 1, 4)",
```

Comment:
The whole point of this formatting with //prettier-ignore is to have the structure look like a real grid. It kinda deafeat the purpose if you add cells that aren't well aligned.

But that's useful for visualising the argument "range" (A1:C3), but we don't really care for the rest. Change the tests A5, B5, C5... to A5, A6, A7... and we should be good

Reply:
Ohh, Okay!!
I've made the suggested changes.